pkg/ingester/owned_series_test.go

```diff
@@ -26,60 +26,67 @@ import (
 
 	"github.com/grafana/mimir/pkg/mimirpb"
 	"github.com/grafana/mimir/pkg/util"
+	"github.com/grafana/mimir/pkg/util/testkafka"
 	"github.com/grafana/mimir/pkg/util/validation"
 )
 
-const ownedServiceTestUser = "test-user"
 const ownedServiceSeriesCount = 10
 const ownedServiceTestUserSeriesLimit = 10000
 
 const defaultHeartbeatPeriod = 100 * time.Millisecond
 
-type ownedSeriesTestContext struct {
+type ownedSeriesTestContextBase struct {
 	seriesToWrite []series
-	seriesTokens  []uint32
-	ingesterZone  string
 
+	user        string
```

Comment:
moved `ownedServiceTestUser` to `TestOwnedSeriesService` test, because `TestOwnedSeriesServiceWithPartitionsRing` needs different user, so that we can test some scenarios for moving series between partitions.